docs/home/contribute/generated-reference/kubernetes-components.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Generating Reference Pages for Kubernetes Components and Tools
+---
+
+{% capture overview %}
+
+This page shows how to use the `update-imported-docs` tool to generate
+reference documentation for tools and components in the
+[Kubernetes](https://github.com/kubernetes/kubernetes) and
+[Federation](https://github.com/kubernetes/federation) repositories.
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+* You need a machine that is running Linux or MacOS.
+
+* You need to have this software installed:
+
+    * [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+    * [Golang](https://golang.org/doc/install) version 1.8 or later
```

Comment:
Just found out kubernetes/kubernetes is now requiring Go v1.9+ in order to run `hack/generate-docs.sh`.